.github/workflows/build-packages.yml

```diff
@@ -26,15 +26,18 @@ jobs:
         run: |
           GOOS=linux GOARCH=amd64 go build -a -o goProbe ./cmd/goProbe
           GOOS=linux GOARCH=amd64 go build -a -o goQuery ./cmd/goQuery
+          GOOS=linux GOARCH=amd64 go build -a -o goConvert ./cmd/goConvert
 
       - name: Deploy artifacts
         run: |
           mkdir -p .debpkg/usr/local/bin
           mkdir -p .debpkg/etc/systemd/system
 
-          # Binaries
+          # Binaries / Tarball
           cp goProbe .debpkg/usr/local/bin/goProbe
           cp goQuery .debpkg/usr/local/bin/goQuery
+          cp goConvert .debpkg/usr/local/bin/goConvert
```

Comment:
Wondering if we should package the legacy tool for the next release only.

Reply:
Yeah, but we can add that one manually on top of that release, IMHO we should automate that, just so we have to clean it up again... :stuck_out_tongue: